packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5167,6 +5184,64 @@ Protocol settings for authentication using the [OpenID Connect Protocol](http://
   }
 }
 ```
+##### OIDC Algorithms object
+
+The `OIDC` protocol supports `request` algorithm and verification settings.
+
+| Property | Description                                                   | DataType                                                                 | Nullable | Readonly |
+| -------- | ------------------------------------------------------------- | ------------------------------------------------------------------------ | -------- | ------   |
+| request  | Algorithm settings used to sign an authorization request | [OIDC Request Algorithm object](#OIDC-request-algorithm-object)   | FALSE    | FALSE    |
+
+```json
+{
+  "protocol": {
+    "type": "OIDC",
+    "algorithms": {
+      "request": {
+        "signature": {
+          "algorithm": "HS256",
+          "scope": "REQUEST"
+        }
+      }
+    }
+  }
+}
+```
+
+###### OIDC Request Algorithm object
+
+Algorithm settings for signing authorization requests sent to the IdP:
+
+| Property  | Description                                                 | DataType                                                                                   | Nullable | Readonly |
+| --------- | ----------------------------------------------------------- | ------------------------------------------------------------------------------------------ | -------- | -------- |
+| signature | Algorithm settings used to sign an authorization request | [OIDC Signature Algorithm object](#oidc-request-signature-algorithm-object) | FALSE    | FALSE    |
+
+```json
+{
+  "protocol": {
+    "type": "OIDC",
+    "algorithms": {
+      "request": {
+        "signature": {
+          "algorithm": "HS256",
+          "scope": "REQUEST"
+        }
+      }
+    }
+  }
+}
+```
+
+###### OIDC request Signature Algorithm object
+
+HMAC Signature Algorithm settings for signing authorization requests sent to the IdP:
+
+| Property    | Description                                                                        | DataType             | Nullable | Readonly |
+| ----------- | ---------------------------------------------------------------------------------- | -------------------- | -------- | -------- |
+| algorithm   | The HMAC Signature Algorithm used when signing an authorization request  | `HS256`, `HS384` or `HS512` | FALSE    | FALSE    |
+| scope       | Specifies whether to digitally sign authorization request to the IdP | `REQUEST` or `NONE`  | FALSE    | FALSE    |
+
+> **Note:** The `algorithm` property is ignored when disabling request signatures (`scope` set as `NONE`).
```

Comment:
```suggestion
> **Note:** The `algorithm` property is ignored when you disable request signatures (`scope` set as `NONE`).
```

Reply:
ok, I updated this. I'm also updating line 4970 to make it consistent